anonymize.py

```diff
@@ -53,19 +54,26 @@ def get_updates(config):
                     updates.append("`%s` = INET_NTOA(RAND()*1000000000)" % field)
             elif operation == 'random_email':
                 for field in listify(details):
-                    updates.append("`%s` = CONCAT(id, '@mozilla.com')"
-                                   % field)
+                    updates.append("`%(field)s` = CONCAT(`%(n)s`, '@inbucket.plurall.net')" % dict(field=field,n=pkName))
             elif operation == 'random_username':
                 for field in listify(details):
-                    updates.append("`%s` = CONCAT('_user_', id)" % field)
+                    updates.append("`%(s)s` = CONCAT('_user_', `%(n)s`)" % dict(s=field,n=pkName))
             elif operation == 'hash_value':
                 for field in listify(details):
-                    updates.append("`%(field)s` = MD5(CONCAT(@common_hash_secret, `%(field)s`))"
-                                   % dict(field=field))
+                    updates.append("`%(field)s` = MD5(CONCAT(@common_hash_secret, `%(field)s`))" % dict(field=field))
             elif operation == 'hash_email':
                 for field in listify(details):
-                    updates.append("`%(field)s` = CONCAT(MD5(CONCAT(@common_hash_secret, `%(field)s`)), '@mozilla.com')"
-                                   % dict(field=field))
+                    updates.append("`%(field)s` = CONCAT(MD5(CONCAT(@common_hash_secret, `%(field)s`)), '@inbucket.plurall.net')" % dict(field=field))
```

Comment:
@williamvalentim  maybe should you change this email "plurall.net" to something generic?